src/Common/Factory/AbstractFactoryRule.php

```diff
@@ -8,7 +8,7 @@
 
 abstract class AbstractFactoryRule extends SprykerAbstractRule
 {
-    const PATTERN_ZED_FACTORY = '/\\\\*\\\\.+\\\\.+\\\\.+Factory$/';
+    const PATTERN_ZED_FACTORY = '/\\\\*\\\\.*\\\\.*\\\\Business\\\\[^\\\\]*Factory$/';
```

Comment:
Why changing .+ to .* ? That makes this non required and is not a good change IMO.

Reply:
oops, missed that. seems like my local sniffer is outdated.